### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by agrimrules**
After a lot of hacking looking around reverse engineering I finally have CI setup. please look at `circle.yml` to get an idea of what is done in order to get build images

### CODE REVIEW

**Comment by agrimrules** [comment]
I squashed everything down to a single commit. Must ensure this is a standard practice moving forward. Also, shall we just ignore the `.idea` directory as a whole?

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,8 @@
 /build
 /captures
 .externalNativeBuild
+*.apk
```
Comment (shoab10) [inline .gitignore:10]: Redundant. since apk s go into /build. I dont know about rest of the extensions you added. But if you are confident. Can go ahead.

**Review by shoab10 — COMMENTED** [review]
Dont know about ignoring .idea . what say?

File: .gitignore (hunk 1 above)
Comment (agrimrules) [reply .gitignore:10]: https://www.gitignore.io/api/android%2Candroidstudio

Is what I used as reference

Comment (shoab10) [reply .gitignore:10]: okay. Let it be then.

**Review by shoab10 — APPROVED** [review]

**Pull request merged**